src/json-patch-duplex.ts

```diff
@@ -85,37 +85,44 @@ module jsonpatch {
   var objOps = {
     add: function (obj, key) {
       obj[key] = this.value;
-      return true;
     },
     remove: function (obj, key) {
+      var removed = obj[key];
       delete obj[key];
-      return true;
+      return {removed: removed};
     },
     replace: function (obj, key) {
+      var removed = obj[key];
       obj[key] = this.value;
-      return true;
+      return {removed: removed};
     },
     move: function (obj, key, tree) {
+      var getOriginalDestination : any = {op: "_get", path: this.path};
```

Comment:
Can't we just use what is returned by apply call from line 107 https://github.com/Starcounter-Jack/JSON-Patch/pull/110/files#diff-48fd3c82ba3fb648fc877536b60e6f9dR109